### ISSUE TITLE
[VersionControl] Error merging branch

### ISSUE THREAD

**Issue by jsuarezruiz** [mono/monodevelop#7117]
Error trying to merge a new created branch into master.

```
ERROR [2019-02-08 13:34:21Z]: Failed to query git status
LibGit2Sharp.AmbiguousSpecificationException: More than one file matches the pathspec '.vs/TipCalc/'. You can either force a literal path evaluation (GIT_STATUS_OPT_DISABLE_PATHSPEC_MATCH), or use git_status_foreach().
at LibGit2Sharp.Core.Proxy.git_status_file (LibGit2Sharp.Core.Handles.RepositoryHandle repo, LibGit2Sharp.Core.FilePath path) [0x00028] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/external/libgit2sharp/LibGit2Sharp/Core/Proxy.cs:2923
at LibGit2Sharp.Repository.RetrieveStatus (System.String filePath) [0x00015] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/external/libgit2sharp/LibGit2Sharp/Repository.cs:1610
at MonoDevelop.VersionControl.Git.GitRepository.GetFilesVersionInfoCore (LibGit2Sharp.Repository repo, MonoDevelop.VersionControl.Git.GitRevision rev, System.Collections.Generic.List`1[T] localPaths, System.Collections.Generic.List`1[T] versions) [0x00019] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/addins/VersionControl/MonoDevelop.VersionControl.Git/MonoDevelop.VersionControl.Git/GitRepository.cs:482
at MonoDevelop.VersionControl.Git.GitRepository.GetDirectoryVersionInfo (MonoDevelop.Core.FilePath localDirectory, System.Collections.Generic.IEnumerable`1[T] localFileNames, System.Boolean getRemoteStatus, System.Boolean recursive) [0x001cc] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/addins/VersionControl/MonoDevelop.VersionControl.Git/MonoDevelop.VersionControl.Git/GitRepository.cs:443`
`at MonoDevelop.VersionControl.Git.GitRepository.OnGetVersionInfo (System.Collections.Generic.IEnumerable`1[T] paths, System.Boolean getRemoteStatus) [0x00002] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/addins/VersionControl/MonoDevelop.VersionControl.Git/MonoDevelop.VersionControl.Git/GitRepository.cs:383
```

> VS bug [#789310](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/789310)

**Comment by jsuarezruiz** [comment]
Duplicated: https://github.com/mono/monodevelop/issues/7119

**Comment by sevoku** [comment]
Fixed with #7142
